pygeo/parameterization/DVGeo.py

```diff
@@ -624,6 +660,68 @@ def addPointSet(self, points, ptName, origConfig=True, **kwargs):
             undeformed or deformed configuration. This should almost
             always be True except in circumstances when the user knows
             exactly what they are doing.
+        coord_xfer : function
+            A callback function that performs a coordinate transformation
+            between the DVGeo reference frame and any other reference
+            frame. The DVGeo object uses this routine to apply the coordinate
+            transformation in "forward" and "reverse" directions to go between
+            the two reference frames. Derivatives are also adjusted since they
+            are vectors coming into DVGeo (in the reverse AD mode)
+            and need to be rotated. We have a callback function here that lets
+            the user to do whatever they want with the coordinate transformation.
+            The function must have the first positional argument as the array that is
+            (npt, 3) and the two optional arguments that must be available are "mode"
```

Comment:
Are they optional if they must be available? Why not just make `mode` and `apply_displacement` positional arguments?

Reply:
as we discussed, rephrased the comment a bit to make it clear they are mandatory, but we decided to keep them as keyword arguments to be explicit with the calls.